### PR TITLE
Cog 1194 parameterize racfid in acceptance

### PR DESCRIPTION
**Pull request by kelfink**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1194

#### Brief feature description
- Instead of hardcoding RACFID in the acceptance tests, pass them in on the spec command, so Jenkins can supply the values differently for each environment.

### CODE REVIEW

**Comment by kelfink** [comment]
retest this please

File: JenkinsDeploy [hunk 1]
```diff
@@ -88,13 +88,13 @@ def integrationAcceptenceTest(environment, version){
     stage('Integration Acceptance Test Chrome') {
       sh "docker-compose up -d --build county-admin-test"
       sleep 60
-      sh "docker-compose exec -T --env TZ=${TZ} --env COGNITO_USERNAME=${COG_USER} --env COGNITO_PASSWORD=${COG_PASS} --env COUNTY_AUTHORIZATION_ENABLED=true --env COUNTY_ADMIN_WEB_BASE_URL=https://web.integration.cwds.io/cap county-admin-test bundle exec rspec spec/acceptance --format documentation"
+      sh "docker-compose exec -T --env TZ=${TZ} --env COGNITO_USERNAME=${COG_USER} --env COGNITO_PASSWORD=${COG_PASS} --env ADD_RACFID=${ADD_RACFID}  --env WRONG_COUNTY_RACFID=${WRONG_COUNTY_RACFID} --env COUNTY_AUTHORIZATION_ENABLED=true --env COUNTY_ADMIN_WEB_BASE_URL=https://web.integration.cwds.io/cap county-admin-test bundle exec rspec spec/acceptance --format documentation"
```
Comment (RamuKammagani-osi) [inline JenkinsDeploy]: Why ADD_RACFID??? Why not VALID_RACFID??

File: spec/acceptance_helper.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@
   if ENV['FIREFOX']
     caps = Selenium::WebDriver::Remote::Capabilities.new(accept_insecure_certs: true)
     browser_options = Selenium::WebDriver::Firefox::Options.new
```
Comment (RamuKammagani-osi) [inline spec/acceptance_helper.rb]: Line 15 here is not required if we are not running in headless browser!

File: JenkinsDeploy (hunk 1 above)
Comment (kelfink) [reply JenkinsDeploy]: I think it's more explicit to say it's the RACFID I will "ADD", but also, its in the "contract" for QA.  https://osi-cwds.atlassian.net/browse/COG-1194 see acceptance criteria.  Also, they've made a request of Ops to add these parameters to the Jenkins environment, and I don't want to delay that.   Later we might have other valid RACFID's and they would have different names, also valid.

File: spec/acceptance_helper.rb (hunk 2 above)
Comment (kelfink) [reply spec/acceptance_helper.rb]: Ok I'll remove that.

File: JenkinsDeploy (hunk 1 above)
Comment (RamuKammagani-osi) [reply JenkinsDeploy]: Sounds fair!

**Pull request merged**